### ISSUE TITLE
Develop and agree on code-style standard

### ISSUE THREAD

**Issue by ehdatheadwire** [peregrine-cms/enhancements#23]
### Original Description
A code-style standard should be agreed upon, starting with javascript code. 

### Notes
- Determined in OG issue that this should be done, but not until fully merged into the develop-sling12 branch

### Reference
**Source:** Felix Puetz
**Date of Entry:** 8/31/20
**Date of Last Revision:** 10/22/20
**OG Link:** https://github.com/headwirecom/peregrine-cms/issues/532
